### ISSUE TITLE
Pressing super keeps openig tty1

### ISSUE THREAD

**Issue by evandrocoan** [hanschen/ksuperkey#18]
I have to press `Ctrl+Alt+7` to come back to the desktop environment.

**Comment by wget** [comment]
@evandrocoan Are you sure you didn't mean `Ctrl+Alt+F7` instead of the `+7`?
Otherwise this might be due to the way your GNU/Linux distribution has configured TTY: https://wiki.archlinux.org/index.php/Getty#Add_additional_virtual_consoles

**Comment by hanschen** [comment]
Also, have you done any other tweaks to your key bindings? It sounds like ksuperkey sends Ctrl+Alt+F1 instead of Alt+F1 in your case.

Some more information about your setup (desktop environment, Linux distribution, etc.) would be helpful.

**Comment by evandrocoan** [comment]
Yes, I mean F7, not 7. 

I use Linux Mint 19.1 with XFCE 4.12
```
evandro@evandro-pc:~$ uname -a
Linux evandro-pc 4.15.0-45-generic #48-Ubuntu SMP Tue Jan 29 16:28:13 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

I had configured it with this:
```
ksuperkey -e 'Super_L=Super_L|m'
```
How can I debug it?

**Comment by hanschen** [comment]
What happens if you press `Super_L+m`?

You can also try to run ksuperkey with the `-d` flag for debug mode.

**Comment by evandrocoan** [comment]
> What happens if you press `Super_L+m`?

It opens the start menu. I had configured it for that. 

On now, I had set `ksuperkey -e 'Super_L=Super_L|m'`, before had actually set `Control_L|i`.

Now, I had set `Super_L|m`, it works fine, and it does not open `tty` anymore.

But before setting `Control_L|i`, I had set `Control_L|Alt_L|0`, which was triggering `tty` some times (randomly). Now, with `Super_L|m`, it did not triggered `tty` so far.

I will try use `-d` debug mode next time, if I get new problems.
